spec/helpers/application_helper/buttons/miq_ae_instance_copy_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe ApplicationHelper::Button::MiqAeInstanceCopy do
+  let(:session) { Hash.new }
+  let(:view_context) { setup_view_context_with_sandbox({}) }
+
+  before do
+    Tenant.seed
+    feature_list = %w(
+      miq_ae_class_edit
+      miq_ae_domain_edit
+      miq_ae_class_copy
+      miq_ae_instance_copy
+      miq_ae_method_copy
+      miq_ae_namespace_edit
+    )
+    user = FactoryGirl.create(:user, :features => feature_list)
+    login_as user
+    @domain = FactoryGirl.create(:miq_ae_domain_disabled)
+    @namespace = FactoryGirl.create(:miq_ae_namespace, :name => 'test_namespace', :parent => @domain)
+    @record = FactoryGirl.create(:miq_ae_class, :name => 'test_class', :namespace_id => @namespace.id)
```

Comment:
maybe it is better to call it similar as factory because it is not apparent what is it usage below